dozer-ingestion/aerospike/src/connector.rs

```diff
@@ -414,18 +420,7 @@ async fn map_event(
     {
         let mut fields = vec![Field::Null; columns_map.len()];
         if let Some((pk, _)) = columns_map.get("PK") {
```

Comment:
This get should be infallible (e.g. just an indexing operation `columns_map["PK"]`). The existence of a primary key is a precondition of this function, and if it doesn't exist, we're in a very weird state and it's alright to just panic here.

Reply:
I think this one would be correct. It would only fail if user want to use PK in ingestion and defines that column.
```
if let Some((pk, _)) = columns_map.get("PK") {
    if let Some(pk_in_key) = pk_in_key {
        fields[*pk] = Field::String(pk_in_key.clone());
    } else {
        return Err(AerospikeConnectorError::PkIsNone(key.clone()));
    }
}
```

Reply:
Yes, you're right